dietpi/dietpi-software

```diff
@@ -12440,7 +12440,7 @@ _EOF_
 
       if [[ -f '/opt/papermc/plugins/Geyser-Spigot.jar' ]] && [[ -f '/opt/papermc/plugins/floodgate-bukkit.jar' ]]
       then
-        Create_Config /opt/papermc/plugins/Geyser-Spigot/config.yml papermc 110
+        Create_Config /opt/papermc/plugins/Geyser-Spigot/config.yml papermc 115
```

Comment:
Where did you tested it to get this long required time? On my VM it took something around 10 seconds. I just thought it would be nice to report the time it took to create the config. For end user probably not important but for developers, when it's about tuning the timeouts, its good to know and review how long it took on the different machines 🙂.

Reply:
It took a minimum of about 104 seconds on my VM, with the only modification being the extra GiB of RAM.  A maximum of about 111.

Reply:
Wow that is long, okay not much harm to allow a longer timeout. This is an interactive-only install so users transparently see what's happening and the counter. Also with the above server start now removed, this one will likely take longer again 😉.